Add changelog entry and pre-requisites for payments/addExternal (#695)
MewsSystems/gitbook-connector-api: operations/payments.md

Adds a new external payment to a bill of the specified customer. An external payment represents a payment that is tracked outside of the system. Note this operation supports [Portfolio Access Tokens](../guidelines/multi-property.md).

> **Pre-requisites:** The external payment type must be enabled by the property in order to accept such payments in their Mews environment. Use `configuration/get` endpoint to check the supported external payment types.

Reviewer: ```suggestion
**Prerequisites:** The external payment type must be enabled by the property in order to accept such payments in their Mews environment. Use [Get configuration](configuration.md#get-configuration) to check which external payment types are supported.
```
